src/components/GamePage/PickPlayerDD.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './PickPlayerDD.css';
+
+export default function PickPlayerDD() {
+  return (
+    <>
+      <div className="dropdown-head">
+        <select className="dropdown ui dropdown">
+          <option value="">Who are you?</option>
+          <option value="1">Sam Trahan</option>
+          <option value="2">JP Angelle</option>
+          <option value="3">Logan Sonnier</option>
```

Comment:
Should Logan be part of it since this is just people that are beating Logan?

Reply:
I know but I also want logan to participate in the leader board so I added him in.

Reply:
Gotcha gotcha. Makes sense.